README.md

```diff
@@ -15,9 +15,27 @@ from scratch as well as keep it up-to-date over time.
 
 ### Prerequisites
 
+* `brew`
```

Comment:
did [this](https://github.com/travi/dotfiles/blob/31a28e535720b186a7b2ecf77b054be62a451742/osx/software.sh#L15-L21) not work? it should install brew as part of the software installation part of setup (which should prompt to install `xcode-select` if its not installed)